### PR TITLE
MAISTRA-1251 Allow overriding Maistra Istio repo in download-charts.sh using env var

### PR DESCRIPTION
**Pull request by luksa**

### CODE REVIEW

**Review by dgn — APPROVED** [review]

**Comment by luksa** [comment]
@dgn I added one small improvement. Can you review the second commit? Thanks.

**Comment by luksa** [comment]
@dgn there are now two small improvements :) sorry about that

File: build/download-charts.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ PLATFORM=linux
 ISTIO_NAME=istio-${ISTIO_VERSION}
 ISTIO_FILE="${MAISTRA_BRANCH}.zip"
 ISTIO_URL="${MAISTRA_REPO}/archive/${MAISTRA_BRANCH}.zip"
-EXTRACT_DIR="istio-${MAISTRA_BRANCH}"
+EXTRACT_DIR="${MAISTRA_REPO##*/}-${MAISTRA_BRANCH}"
```
Comment (dgn) [inline build/download-charts.sh:22]: that's some bash magic right there

**Review by dgn — APPROVED** [review]
double-approve

File: build/download-charts.sh (hunk 1 above)
Comment (dgn) [reply build/download-charts.sh:22]: nit: I generally prefer explicit, e.g. `$(basename ${MAISTRA_REPO})`

**Review by rcernich — APPROVED** [review]

**Pull request merged**